### PR TITLE
Add travis bits

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+sudo: false
+before_install:
+- npm install -g origami-build-tools
+script:
+- make install && make test && make provision
+- make tidy
```
Comment (ironsidevsquincy) [inline .travis.yml]: why not do

```
install
- npm install origami-build-tools && make install
script
- make test && make provision
- make tidy
```

saves on the extra `npm install` that would happen by default if no `script` step supplied

File: Makefile [hunk 2]
```diff
@@ -5,7 +5,7 @@ TEST_URL := "http://ft-styleguide-demoer-br-${GIT_HASH}.herokuapp.com/__gtg"
 
 install:
 	obt install --verbose
-	$(foreach var, $(COMPONENTS), cd bower_components/$(var); bower install; obt build --js=./demos/src/main.js;)
+	$(foreach var, $(COMPONENTS), cd bower_components/$(var); npm install -g origami-build-tools && bower install && obt build --js=./demos/src/main.js;)
```
Comment (ironsidevsquincy) [inline Makefile]: wont obt already be installed (or previous step would fil_?

File: .travis.yml (hunk 1 above)
Comment (keirog) [inline .travis.yml]: It was a blind and desperate attempt to get around a build error that I didn't understand :-)

File: Makefile (hunk 2 above)
Comment (keirog) [inline Makefile]: Please see previous confession

**Pull request merged**